.gitignore

```diff
@@ -11,3 +11,4 @@ pom.xml.asc
 .lein-failures
 /resources/public/
 node_modules/
+/resources/templates/.sass-cache
```

Comment:
I think I've been checking in the .sass-cache so far. Think we should keep it out?

Reply:
Actually, I should have checked to see that you were committing this.  I just assumed it was newly created on my system.  Habit of excluding things named `*cache*` I guess :)

I don't know anything about SASS, so if it's safe and common to keep that, I'll back that exclude out.

Reply:
Doesn't hurt - let's just keep it for now. This is kind of a cobbled-togther asset pipeline. (I added JS bundling separately using Webpack last week).